### PR TITLE
Destroy Artificial Moon building on use

### PR DESCRIPTION
**Pull request by MichaelCourtney**
https://github.com/freeorion/freeorion/issues/3973

### CODE REVIEW

File: default/scripting/buildings/ART_MOON.focs.txt [hunk 1]
```diff
@@ -12,14 +12,16 @@ BuildingType
     ]
     enqueuelocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
     effectsgroups = [
-        [[SPECIES_LIKES_OR_DISLIKES_BUILDING_STABILITY_EFFECTS]]
```
Comment (geoffthemedio) [inline default/scripting/buildings/ART_MOON.focs.txt:15]: intentional? should be noted in commit message...

Comment (MichaelCourtney) [reply default/scripting/buildings/ART_MOON.focs.txt:15]: It was consistent with Gaia and also with colony buildings, terraforming etc.
I can add it into the commit message

Comment (Grummel7) [reply default/scripting/buildings/ART_MOON.focs.txt:15]: I've had this problem before when working on terraforming. Now I think we should rather suggest to remove it from this from all temporary building.

**Comment by geoffthemedio** [comment]
could you write the multi-part commit message with the two things on separate lines?

**-Destroy building on creation of special
-Remove Building Stability Macro from Artificial Moon**

**Pull request merged**